packages/api/bin/serve.js

```diff
@@ -250,7 +271,44 @@ async function serveDistributionApi(stackName = 'localrun', done) {
   return distributionApp.listen(port, done);
 }
 
+/**
+ * Removes all additional data from tables and repopulates with original data.
+ *
+ * @param {string} user - defaults to local user, testUser
+ * @param {string} stackName - defaults to local stack, localrun
+ */
+async function resetTables(user = 'testUser', stackName = 'localrun') {
+  if (inTestMode()) {
+    setTableEnvVariables(stackName);
+    process.env.system_bucket = 'localbucket';
+    process.env.stackName = stackName;
+
+    // Remove all data from tables
+    const providerModel = new models.Provider();
+    const collectionModel = new models.Collection();
+    const rulesModel = new models.Rule();
+    const executionModel = new models.Execution();
+    const granulesModel = new models.Granule();
+    const pdrsModel = new models.Pdr();
+
+    try {
+      await rulesModel.deleteRules();
```

Comment:
We can wrap all of these delete calls in `Promise.all` to run all of these in parallel

Reply:
Yes I can wrap most of them. Rules does need to be deleted first because collections and providers will error if they have an associated rule.